docs/release-notes.md

```diff
@@ -2,6 +2,21 @@
 
 This page summarizes changes for each major update in Proton and Timeplus Cloud, including new features and important bug fixes.
 
+## Dec 11, 2023
+
+*Proton:*
+  * (Preview Feature) New datasources: NATS and NATS JetStream. The configuration UI will be available soon in Timeplus Cloud. Contact us if you'd like to try this feature.
```

Comment:
This is nothing to do with Proton. It's implemented in Neutron and only available in Timeplus Cloud/Platform.